packages/text-document-utils/src/lineOffsets.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
File name does not match the naming convention. Please use kebab-case.

Reply:
done.